src/generators/randgraphs.jl

```diff
@@ -598,6 +598,28 @@ function random_regular_digraph(n::Integer, k::Integer; dir::Symbol=:out, seed::
     end
 end
 
+@doc_str """
+    random_tournament_digraph(n)
+
+Create a random directed [tournament graph]
+(https://en.wikipedia.org/wiki/Tournament_(graph_theory))
```

Comment:
remove line break to make valid markdown

Reply:
This line break will be automatically ignored when `Documenter.jl` is generating documentations. I put line break here because otherwise it makes too long a line.

Reply:
Finally, escape the url:  `[tournament graph](https://en.wikipedia.org/wiki/Tournament_%28graph_theory%29)` to make the link actually working (otherwise the closing parentheses are parsed wrongly)

Reply:
Ping

Reply:
Yup, this needs to be done.

Reply:
Thanks! Modified.